Ginger/Ginger/App.config

```diff
@@ -2,7 +2,17 @@
 <configuration>
 	<configSections>
 	</configSections>
+	<appSettings>
+		<add key="TraceActivateSwitch" value="True"/>
+		<add key="TraceLevelSwitch_Error" value="True"/>
+		<add key="TraceLevelSwitch_Info" value="True"/>
+		<add key="TRACE_LOG_FILES_PATH" value="C:\ALMlogs"/>
```

Comment:
Please make sure it is generating logs in Ginger log folder

Reply:
Done